assets/css/styles.css

```diff
@@ -0,0 +1,376 @@
+/* 
+
+Colour Scheme Variables 
+
+*/
+
+:root {
+    --clr-background: #fcf7f8;
+    --clr-nav: #5B616A;
+    --clr-footer: #60E1E0;
+    --clr-denim: #275DAD;
+    --clr-hover: #60E1E0;
+    --clr-blue-yonder: #ABA9C3;
+}
+
+
+/* 
+
+General styles for web page 
+
+*/
+
+
+/* typography and background colour for web page */
+
+body {
+    font-family: Verdana, sans-serif;
+    background: var(--clr-blue-yonder)
+}
+
+.heading {
+    padding: 15px 35px;
+    border-bottom: 5px solid var(--clr-hover);
+    letter-spacing: 2px;
+}
+
+.heading h1 {
+    display: flex;
+    justify-content: flex-end;
+    font-size: 30px;
+}
+
+
+/* image fading for project cards */
+
+.img-opacity {
+    opacity: 0.6;
+}
+
+.img-opacity:hover {
+    opacity: 1.0;
+    transition: all 200ms ease-in;
+}
+
+
+/* 
+
+Navigation Styles 
+
+*/
+
+
+/* fixed navigation bar */
+
+.fixed {
+    position: fixed;
+    top: 0;
+    width: 100%;
+    z-index: 1;
+}
+
+.nav-list {
+    padding: 21px;
+    background: var(--clr-nav);
+    box-shadow: 0px 0px 50px var(--clr-nav);
+    display: flex;
+    flex-wrap: wrap;
+    width: 100%;
+    align-items: center;
+}
+
+.logo {
+    margin-right: auto;
+}
+
+.logo a {
+    text-decoration: none;
+    align-items: center;
+    font-size: 15px;
+    color: var(--clr-background);
+    letter-spacing: 4px;
+}
+
+
+/* hamburger menu */
+
+#toggle {
+    display: block;
+    cursor: pointer;
+    padding-bottom: 7px;
+}
+
+
+/* navigation button container */
+
+#items {
+    display: flex;
+    flex-wrap: wrap;
+}
+
+
+/* navgiation buttons/id's for script.js */
+
+#navItem1,
+#navItem2,
+#navItem3 {
+    width: 100%;
+    text-align: center;
+    display: none;
+}
+
+#navItem1 a,
+#navItem2 a,
+#navItem3 a {
+    text-decoration: none;
+    font-size: 15px;
+    color: var(--clr-background);
+    letter-spacing: 3px;
+}
+
+
+/* colour for hovering cursor over each anchor element */
+
+#navItem1 a:hover,
+#navItem2 a:hover,
+#navItem3 a:hover,
+.logo a:hover {
+    color: var(--clr-hover);
+    transition: all 200ms ease-in;
+}
+
+
+/* dropdown list styling for navigation buttons*/
+
+#navItem1.active {
```

Comment:
I think you mean to use the pseudo class `:active` here, rather than `.active` which would signal a class selector (same with the next group too)